deployments/properties/azure-kubernetes-service-helm.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Build and Deploy an app to AKS with Helm",
```

Comment:
This name looks big, can we have smaller name for eg: `Deploy to AKS with Helm`

Reply:
Shortened